packages/date-picker/src/_primitives/Calendar/Calendar.module.scss

```diff
@@ -263,7 +263,7 @@ $rdp-cell-size: 40px;
   -moz-appearance: none;
   -webkit-appearance: none;
   appearance: none;
-  position: absolute !important;
+  position: absolute;
```

Comment:
This breaks the linter after taking it out of global styles. I removed `!important` and checked it out in storybook locally, and seemed to behave the same without it. `position: absolute` still ends up in the class rule & doesn't seem to get overridden by anything